### PR TITLE
Turuta_homework_6

### PR DESCRIPTION
**Pull request by turuta7**

### CODE REVIEW

**Comment by markoffden** [comment]
and task1 ?

**Comment by turuta7** [comment]
> and task1 ?
Forgot to add :(

File: homeworks/Aleksandr.Turuta_turuta7/homework_6/1.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+Object.defineProperties(user, {
+    fullName: {
+        enumerable: true,
+        configurable: true,
+        writeble: true,
+        set: (name) => {
+            this.fullName = name
+                .replace(/([[A-Za-z])\S/g, s => s.toLowerCase())
+                .replace(/(^|\s|[-])\S/g, s => s.toUpperCase());
+        },
```
Comment (markoffden) [inline homeworks/Aleksandr.Turuta_turuta7/homework_6/1.js]: looks a bit complicated... you could check for every first letter in a word `\b\w` for example

**Comment by markoffden** [comment]
accepted, but using regexp could be optimized

**Pull request merged**